content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -540,11 +540,11 @@ spec:
 
 ### Access Modes
 
-Claims use the same conventions as volumes when requesting storage with specific access modes.
+Claims use [the same conventions as volumes](#access-modes) when requesting storage with specific access modes.
 
 ### Volume Modes
```

Comment:
```suggestion
### Volume Modes {#volume-mode-pvc}
```
will set a custom fragment identifier (avoiding potential for a collision / confusion)